lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -627,12 +647,14 @@ def write(chunk)
                 log_name: log_name,
                 resource: group_level_resource,
                 labels: group_level_common_labels,
+                partial_success: @partial_success,
                 entries: entries)
             entries_count = entries.length
-
-            # TODO: RequestOptions
             begin
-              client.write_entry_log_entries(write_request)
+              client.write_entry_log_entries(
+                write_request,
+                options: { api_format_version: '2' }
+              )
             rescue Google::Apis::Error => error
```

Comment:
Let's add a blank line before this `rescue`, for consistency.

Reply:
This one and the rest are not at the same level though. It might be more confusing that way.